app/views/books/show.html.erb

```diff
@@ -13,21 +13,46 @@
   <hr>
   <div class="container">
     <h1>Locations:</h1>
+    <%# accordion %>
     <% if @locations_uniq.empty? %>
       <p><em>Sorry, but this book is currently not available...</em></p>
     <% else %>
+      <%# iterate over available locations %>
       <% @locations_uniq.each do |location| %>
-        <div class="container mb-3">
-          <h4 class="m-0"><%= location.name %></h4>
-          <%# show users that offer the book at this location %>
-          <p class="m-0">Set up a meeting with:</p>
-          <% location.users.each do |user| %>
-            <% if (user.copies.find_by book_id: params[:id]) %>
-              <% unless user == current_user %>
-                <%= link_to user.username, new_meeting_path(location: location, copy: (user.copies.find_by book_id: params[:id])) %>
-              <% end %>
-            <% end %>
-          <% end %>
+        <div class="accordion accordion-flush">
+          <div class="accordion-item mt-2">
+            <div class="accordion-header" id="flush-heading<%= location.id %>">
+              <div class="button collapsed card-location text-white"
+                    data-bs-toggle="collapse"
+                    data-bs-target="#flush-collapse<%= location.id %>"
+                    aria-expanded="false"
+                    aria-controls="flush-collapse<%= location.id %>"
+                    style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url(https://images.unsplash.com/photo-1507550284711-08dadbf8621f?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8N3x8Ym9vayUyMGNhZmV8ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=900&q=60)"
```

Comment:
I get the idea, I think. Maybe it would be clearer to put the style part (gradient) in the scss as an overlay (if needed) and then a set of some image links into a `location.image` attribute for every location that is added in the controller for now. This way, the view code would be prepared for later.